ming_sim/audience_night.py

```
@@ -849,7 +851,10 @@ def open_night(
         # #657 P7/W1：垫位路径只许空 body；不叠复命场面、不用固定开夜句。
         open_body = ""
     else:
-        open_body = body or f"{location}·{time_of_day}，召对启。"
+        # N1（#1561）：删除固定 opening fallback。空 body 时留空垫位；
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)